homeassistant/components/foscam/camera.py

```diff
@@ -57,6 +58,8 @@ def __init__(self, device_info):
         self._foscam_session = FoscamCamera(
             ip_address, port, self._username, self._password, verbose=False)
 
+        self._media_port = self._foscam_session.get_port_info()[1]['mediaPort']
```

Comment:
Will the result of `get_port_info` always be an array in which the 2nd element contains the media Port ?

Reply:
As far as I can tell it's always returned as `(status_code, params)` with params being a dictionary that only contains webPort, mediaPort, httpsPort and onvifPort. I can't tell if the dictionary is ordered.

Reply:
ok perfect.